cmd/geth/main.go

```diff
@@ -173,10 +174,10 @@ var (
 		utils.HTTPListenAddrFlag,
 		utils.HTTPPortFlag,
 		utils.HTTPCORSDomainFlag,
-		utils.AuthListenFlag,
-		utils.AuthPortFlag,
-		utils.AuthVirtualHostsFlag,
-		utils.JWTSecretFlag,
+		// utils.AuthListenFlag,
```

Comment:
Why not remove them?

Reply:
comment them express that they are disabled on purpose, comparing to go-ethereum